### DISCUSSION TITLE
Pre-releases

### DISCUSSION THREAD

**Discussion by erlend-sh** [pintariching/rustle#41] [Ideas]
What would qualify as a v0.0.1 pre-release?

I think it would benefit rustle to get a pre-release out on GitHub every few weeks, even if there’s very limited functionality to play around with.

Rustle is actively moving forward, and it stands a good chance of attracting new contributors as long as it regularly reminds people in the Rust & Svelte communities of its ongoing development and rudimentary roadmap.

**Comment by qwertydelle** [comment]
I think once parsing is implemented that could qualify as a pre release.

**Reply by pintariching** [reply]
I agree, once we get the reading, parsing and generating the AST going, I think we could qualify for a pre-release.

**Comment by pintariching** [comment]
I think we could update the README with a roadmap, I'll open an issue if anybody wants to work on it.

**Comment by erlend-sh** [comment]
Resolved in #58

**Comment by erlend-sh** [comment]
Thanks to #65 perhaps it’s time to release v0.2.0?

**Comment by qwertydelle** [comment]
I say we polish it out a bit and release something that could generate fully usable svelte code.
